db/migrate/20150622182740_create_people.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePeople < ActiveRecord::Migration
+  def change
+    create_table :people do |t|
+      t.string  :first_name, null: false
+      t.string  :last_name
+      t.integer :task_id
```

Comment:
This column is unnecessary as the `tasks` table has a `user_id` column (created via the `belongs_to :person` call) and the direction of the association, with people having many tasks, means that you need to use the foreign key on the tasks so that each user can have multiple tasks, rather than each task having multiple users.

Not sure that's the best explanation, feel free to ask follow up questions if that doesn't make sense or chat in person next time I'm in the classroom.